packages/utils/test/envelope.test.ts

```diff
@@ -1,16 +1,82 @@
-import type { Event, EventEnvelope } from '@sentry/types';
+import type { Event, EventEnvelope, SpanAttributes } from '@sentry/types';
 
+import {
+  SEMANTIC_ATTRIBUTE_EXCLUSIVE_TIME,
+  SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_UNIT,
+  SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_VALUE,
+  spanToJSON,
+} from '@sentry/core';
+import { SentrySpan } from '@sentry/core';
 import {
   addItemToEnvelope,
   createEnvelope,
+  createSpanEnvelopeItem,
   forEachEnvelopeItem,
   parseEnvelope,
   serializeEnvelope,
 } from '../src/envelope';
+import { dropUndefinedKeys } from '../src/object';
 import type { InternalGlobal } from '../src/worldwide';
 import { GLOBAL_OBJ } from '../src/worldwide';
 
 describe('envelope', () => {
+  describe('createSpanEnvelope()', () => {
+    it('span-envelope-item of INP event has the correct object structure', () => {
+      const attributes: SpanAttributes = dropUndefinedKeys({
+        release: 'releaseString',
+        environment: 'dev',
+        transaction: '/test-route',
+        [SEMANTIC_ATTRIBUTE_EXCLUSIVE_TIME]: 80,
+        user: 10,
+        profile_id: 'test-profile-id',
+        replay_id: 'test-replay-id',
+      });
+
+      const startTime = 1713365480;
+
+      const span = new SentrySpan({
+        startTimestamp: startTime,
+        endTimestamp: startTime + 2,
+        op: 'ui.interaction.click',
+        name: '<unknown>',
+        attributes,
+      });
+
+      span.addEvent('inp', {
+        [SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_UNIT]: 'millisecond',
+        [SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_VALUE]: 100,
+      });
+
+      const spanEnvelopeItem = createSpanEnvelopeItem(spanToJSON(span));
+
+      const expectedObj = {
+        data: {
+          'sentry.origin': expect.any(String),
+          'sentry.op': expect.any(String),
+          release: expect.any(String),
+          environment: expect.any(String),
+          transaction: expect.any(String),
+          'sentry.exclusive_time': expect.any(Number),
+          user: expect.any(Number),
+          profile_id: expect.any(String),
+          replay_id: expect.any(String),
+        },
+        description: expect.any(String),
+        op: expect.any(String),
+        span_id: expect.any(String),
+        start_timestamp: expect.any(Number),
+        timestamp: expect.any(Number),
+        trace_id: expect.any(String),
+        origin: expect.any(String),
+        exclusive_time: expect.any(Number),
+        measurements: { inp: { value: expect.any(Number), unit: expect.any(String) } },
+      };
+
+      expect(spanEnvelopeItem[0].type).toBe('span');
```

Comment:
@cleptric I wonder, should we send an `otel_span` here instead? Since we discussed yesterday that SDKs should not longer send `span` items going forward.

Reply:
`INP` and the upcoming `HTTP.CLIENT` spans are the only exception, which only applies to the JS SDK. I'm afraid we would also delay the v8 release too much, so I think it's the pragmatic solution to move forward here.